backend/spec/features/admin/homepage_spec.rb

```diff
@@ -51,6 +55,20 @@
         within('.selected .admin-subnav') { page.find_link("Prototypes")['/admin/prototypes'] }
       end
     end
+
+    context "visiting the promotions tab" do
+      before(:each) do
+        visit spree.admin_promotions_path
+      end
+
+      it "should have a link to promotions" do
+        within('.selected .admin-subnav') { page.find_link("Promotions")['/admin/promotions'] }
```

Comment:
I don't think this `[]` syntax works (I see it elsewhere in the file, I don't think that does anything either).

```
page.find_link("Promotions")['/admin/promotions'] #=> nil, no spec failure
page.find_link("Promotions")['foooooooooo'] #=> nil, no spec failure
```

Instead can we make this

```
expect(page).to have_link("Promotions")
```